arangodb-net-standard/ApiErrorException.cs

```diff
@@ -12,7 +12,7 @@ public ApiErrorException()
         {
         }
 
-        public ApiErrorException(ApiErrorResponse error) : base(error.ErrorMessage)
+        public ApiErrorException(ApiErrorResponse error) : base(error == null ? string.Empty : error.ErrorMessage)
```

Comment:
Should we comment on the `ApiError` property that the property can be null if ArangoDB returns an empty content in the response? I'm not sure under which circumstences it can happen though. Not a blocker.

Reply:
Thanks. I've added the comments.